layers/meta-resin-qemu/recipes-core/images/resin-image.bbappend

```diff
@@ -0,0 +1,31 @@
+IMAGE_FSTYPES_append_qemuall = " resin-sdcard"
+
+# Customize resin-sdcard
+RESIN_IMAGE_BOOTLOADER_qemuall = "grub"
```

Comment:
having here qemuall (and for all above) is wrong as this includes qemuarm which obviously doesn't support grub. We need to replace this by two line of qemux86/qemux86-64.